Makefile

```diff
@@ -168,7 +168,7 @@ docker-doctests-fast: docker-build
 	    || true
 	@echo "Running short doctests..."
 	@docker run --name ${CONTAINER_NAME} --rm -it ${IMAGE_NAME} sh -c "\
-          pytest --skip-long-doctests  --doctest-modules -n auto -vv -s \
+          pytest --skip-long-doctests  --doctest-modules -n auto -vv \
```

Comment:
@Dioprz  wrote at one point, that the `-s` flag does something important. But I dont remember what.

Reply:
Yes, it shows the prints produced by the tests (so it's not captured by pytest). However, it is only needed in the KAT-tests command, so this is okey!